WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2434,3 +2434,22 @@ PrefCalWorkHoursCell    = padding:0 @PrefCalInternalSpacing@;
 PrefCalWorkHoursOption  = margin-bottom:@PrefCalInternalSpacing@;
 PrefCalWorkHoursValue   = padding-left:@PrefViewWrapperPadding@;
 PrefCalCustomWHCells    = padding:@PrefCalInternalSpacing@;
+
+######################################
+# Preference- Filters interface
+######################################
+
+PrefFilterPage                  = 
+PrefFilterTabView               = margin:@PrefViewInnerSpacing@ 0 0 0;
+PrefFilterTabBar                = @PanelBorder@; @MediumRoundCorners@; @blRadius@:0px; @brRadius@:0px;
+PrefFilterViewToolbar           = margin:@PrefViewWrapperPadding@ 0 0 0;
+PrefFiltersList                 = @PanelBorder@; border-top:none; @MediumRoundCorners@; @tlRadius@:0px; @trRadius@:0px;
+PrefFiltersListEmpty            = padding:@SkinWrapperPadding@;
+PrefFilterCellLayout            = display:flex; justify-content:space-between; align-items:center;
```

Comment:
it should not break drastically in IE 11, some small minor issues can be ignored for now and fixed later